src/components/search/__tests__/search.test.js

```diff
@@ -119,4 +119,20 @@ describe('search', () => {
       expect(tree).toMatchSnapshot();
     });
   });
+
+  describe(testCases.withConnector.description, () => {
+    let testCase;
+    let wrapper;
+    let tree;
+
+    beforeEach(() => {
+      testCase = testCases.withLayout;
+      wrapper = renderer.create(testCase.element);
```

Comment:
We'll want adjust these lines to pull in the `withConnector` test case and also since the `withConnector` test case is formatted as props (not an element like the `withLayout` test case) we'll need to use React's `createElement` to build the `wrapper`.

```
testCase = testCases.withConnector;
wrapper = renderer.create(
  React.createElement(testCase.component, testCase.props)
);
```

After you make this change, you'll need to update the jest snapshots with `jest -u` and you can commit those files. The diff for this test case will look lengthy, but that's only because we're removing the generated snapshot code that got pulled in from the `withLayout` test case.

Reply:
oops!